kytos/core/controller.py

```diff
@@ -382,6 +382,8 @@ def start_controller(self):
 
         task = self.loop.create_task(self.api_server.serve())
         self._tasks.append(task)
+        while not self.api_server.server.started:
```

Comment:
Great solution, I'm glad this wait loop and this uvicorn `started` flag worked. Can  you extract this as a private method though? Something like `_wait_api_server_started()` just so it's easier to also test and document in the docstring why we have it.